src/App.jsx

```diff
@@ -1,16 +1,12 @@
-import { useState } from 'react';
+import { TodoList  } from './components/TodoList';
 import './App.css';
-import { TodoWrapper  } from './components/TodoWrapper';
 
 function App() {
-  
-
   return (
-    
     <div>
-    <TodoWrapper />
-    </div> 
+      <TodoList />
+    </div>
   );
-};
+}
```

Comment:
suggestion, rename this component from `<TodoWrapper />` to `<Todolist />` since this is , in fact, the TODO list solution